src/common/sha256.cpp

```diff
@@ -0,0 +1,50 @@
+#include "common/sha256.h"
+
+#include "common/exception/runtime.h"
+
+namespace kuzu {
+namespace common {
+
+SHA256::SHA256() {
+    mbedtls_sha256_init(&shaContext);
+
+    if (mbedtls_sha256_starts(&shaContext, false)) {
+        throw RuntimeException{"SHA256 Error"};
```

Comment:
Add a test case to cover this case.

Reply:
It seems like I cannot trigger these errors by choosing a specific string for the e2e test. These errors would only occur if there's an issue with the ```shaContext``` which is wrapped inside ```SHA256```, or with the ```mbedtls``` library itself. Do you have any suggestion for this?

Reply:
Ok, can you leave a comment?